inventory/group_vars/all.yml

```diff
@@ -15,15 +15,15 @@ ssh_pubkey: "{{ lookup('file', '~/.ssh/{{ ssh_pubkey_name }}') }}"
 protocol: https
 
 git_repo: https://github.com/openfoodfoundation/openfoodnetwork.git
-git_version:  v1.8.6
+git_version:  v1.10.1
 
 l10n_git_version: HEAD
 
 
 #----------------------------------------------------------------------
 # Default values (should be overridden in each server config)
 rails_env: staging
-admin_email: "admin@{{ domain }}"
+admin_email: "hello@{{ domain }}"
```

Comment:
This change would affect all the instances, that's why is in `all.yml` file. Any specific reason to change it? You should be able to override it in your specific `uk.yml` file. Makes sense? Can you please try that?